lib/vsc/install/shared_setup.py

```diff
@@ -1047,7 +1055,7 @@ def finalize_options(self):
 
         def _print(self, cmd):
             """Print is evil, cmd is list"""
-            print ' '.join(cmd)
+            print(' '.join(cmd))
```

Comment:
add the future print_function import to this module, since we're doing print here.